test/compress/collapse_vars.js

```diff
@@ -1209,3 +1207,42 @@ collapse_vars_regexp: {
         })();
     }
 }
+
+keep_assign: {
```

Comment:
I see what you've done. Please keep the original test names in the original place. Just change `unused` to `2` in the original tests that have different results.